.github/workflows/main.yml

```diff
@@ -20,6 +20,9 @@ jobs:
       - name: Run tests
         working-directory: ./camera/auth
         run: make test
+      - name: Run lint
+        working-directory: ./camera/auth
+        run: make lint
```

Comment:
Should we run the linter before we run any tests? It looks like tests take ~1min, we would get quicker feedback if we ran the linter before our tests

Reply:
Vscode shows the linter results already, I think it's fine to leave as is. I'm not sure how important quick feedback is here.

Reply:
Tests passing is also a higher priority compared to linting. Github actions stop executing commands on a failure iirc.